modules/jupyter/main.tf

```diff
@@ -125,10 +151,7 @@ resource "helm_release" "jupyterhub" {
       k8s_service_account = var.gcp_and_k8s_service_account
     })
   ]
-  depends_on = [
-    module.iap_auth,
-    module.jupyterhub-workload-identity
-  ]
+  depends_on = [ module.jupyterhub-workload-identity ]
 }
 
 # Need to re-apply: fetch service_id from deployed service
```

Comment:
Below blocks are not required, as the purpose was to fetch backendConfig id, as referred on line 118 (service_id)